app/models/comment.rb

```diff
@@ -88,6 +88,7 @@ class Comment < ApplicationRecord
 
   scope :eager_load_serialized_data, -> { includes(:user, :commentable) }
   scope :good_quality, -> { where("score > ?", LOW_QUALITY_THRESHOLD) }
+  scope :hidden_by_score, -> { where("score <= ?", HIDE_THRESHOLD) }
```

Comment:
I think `below_score_threshold` is a more fitting name but look like this method isn't used anywhere?

Reply:
Yeah, I made a lot of iterations before the final solution, so kept it unintentionally, will remove.